airbyte-webapp/src/pages/SourcesPage/pages/CreateSourcePage/components/SourceForm.tsx

```diff
@@ -54,10 +54,9 @@ const SourceForm: React.FC<IProps> = ({ onSubmit, sourceDefinitions, error, hasS
       afterSelectConnector();
     }
 
-    analyticsService.track("New Source - Action", {
-      action: "Select a connector",
-      connector_source_definition: connector?.name,
-      connector_source_definition_id: sourceDefinitionId,
+    trackNewSourceAction("Select a connector", {
+      connector_source: connector?.name,
+      connector_source_id: sourceDefinitionId,
```

Comment:
The PM's spec for these events calls this event `connector_source_destination_id`. However, it's evenly split with both names across the codebase. For now, the name stays this way, but I will create an issue to make this consistent.